mobile/src/main.jsx

```diff
@@ -17,24 +15,26 @@ import { loadState } from './lib/localStorage'
 import { configureStore } from './lib/store'
 import { initService } from './lib/init'
 import { startBackgroundService, stopBackgroundService } from './lib/background'
-import { initBar, isClientRegistered, resetClient, refreshFolder, onError } from './lib/cozy-helper'
+import { initBar, isClientRegistered, resetClient, startRepeatedReplication, startFirstReplication, onError } from './lib/cozy-helper'
 import { pingOnceADay } from './actions/timestamp'
 
 const renderAppWithPersistedState = persistedState => {
   const store = configureStore(persistedState)
   initService(store)
 
   function requireSetup (nextState, replace, callback) {
-    const client = store.getState().mobile.settings.client
-    const isSetup = store.getState().mobile.settings.authorized
+    const state = store.getState()
+    const client = state.settings.client
+    const isSetup = state.mobile.settings.authorized
+    const isFirstReplicationFinished = state.settings.firstReplication
     if (isSetup) {
       isClientRegistered(client).then(clientIsRegistered => {
         if (clientIsRegistered) {
-          const options = {
-            onError: onError(store.dispatch, store.getState),
-            onComplete: refreshFolder(store.dispatch, store.getState)
+          if (isFirstReplicationFinished) {
```

Comment:
Maybe it won't be this scope's responsibility to know if it starts *the first* replication or *a repeated* replication.
Maybe it may delegate that decision to the lib with a `function startReplication (onError, onComplete)` and implements the decision to be taken in the lib file.